qmp/libvirtrpc/hypervisor_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2016 The go-qemu Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package libvirtrpc is a pure Go implementation of the libvirt RPC protocol.
```

Comment:
Only need a doc comment in one file.  Perhaps we should just standardize on making a `doc.go`.